src/styles/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  transition: $animation-transition;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $dark-grey;
```

Comment:
The variable `$dark-grey` is used here but not defined in this file. Ensure it is defined in an imported file or within this SCSS file.